Gemfile.lock

```diff
@@ -1,16 +1,16 @@
 PATH
   remote: .
   specs:
-    rspec-buildkite-analytics (0.2.0)
-      activesupport
-      rspec-core
-      rspec-expectations
-      websocket
+    rspec-buildkite-analytics (0.3.2)
+      activesupport (>= 5.2, <= 7.0)
+      rspec-core (~> 3.10)
+      rspec-expectations (~> 3.10)
+      websocket (~> 1.2)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.1.4)
+    activesupport (6.1.4.1)
```

Comment:
were these meant to change?

Reply:
yup `Gemfile.lock` change is fine for our gem development. It does not affect users of this gem. It is also possible to have gem without `Gemfile.lock`.